### PR TITLE
#2445 - add Download/Delete buttons and display image size in the image viewer

### LINKED ISSUE #2445: Image download/delete options are not available on mobile
### Problem

When an image is attached to a message in a chatroom. 'Download' and 'Delete' menus are revealed on desktop when a pointer hovers over it. But these are not available on mobile because when user taps on the image, the viewer immediately pops out.

These menus and also the file size information should be added to the image viewer.

### Solution
Add buttons for these functionalities to the image viewer.

### PR DESCRIPTION
**Pull request by SebinSong**
closes #2445 

**[screenshot]**

**[screenshot in forced-color mode]**

---
The issue was that these functionalities are not available on mobile but I thought they are good additions for desktop size screen too and so made them available for both.

Thanks,

### CODE REVIEW

File: frontend/views/containers/chatroom/file-attachment/ChatAttachmentPreview.vue [hunk 1]
```diff
@@ -227,14 +239,18 @@ export default {
     openImageViewer (objectURL) {
       if (!objectURL) { return }
 
+      console.log('!@# attachment list: ', this.attachmentList)
```
Comment (taoeffect) [inline frontend/views/containers/chatroom/file-attachment/ChatAttachmentPreview.vue]: Stray debugging

**Review by taoeffect — CHANGES_REQUESTED** [review]
Seems to work really well!

Fantastic work @SebinSong! One minor issue found.

**Comment by SebinSong** [comment]
@taoeffect Updated the PR again.

**Comment by SebinSong** [comment]
@taoeffect I think this issue has been resolved by [this](https://github.com/okTurtles/group-income/pull/2357) recently merged PR and we can discard this PR.

**Comment by SebinSong** [comment]
@taoeffect as reported in [this Slack thread](https://okturtles.slack.com/archives/C02QF71TK1Q/p1733961421987199), upload-image functionality is currently broken in `master`. Pls don't review this PR until this is resolved.

**Comment by SebinSong** [comment]
@taoeffect Apparently, the broken image upload functionality requires us to refactor the entire image compression logic. Let me make this fix as part of this PR and let you know if it's ready for review again.

**Comment by SebinSong** [comment]
@taoeffect This PR is ready for review again.

**Review by taoeffect — APPROVED** [review]
Brilliant!

**Pull request merged**